Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -3692,6 +3697,10 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
     bool m_hasActiveAnimatedScroll { false };
     bool m_registeredForFullSpeedUpdates { false };
 
+#if ENABLE(MODEL_ELEMENT)
+    bool m_hasModelElement { false };
```

Comment:
Do we also need to reset this state in `WebPageProxy::resetStateAfterProcessExited`?